README.md

```diff
@@ -72,7 +73,7 @@ In `hardhat.config.ts`, you can set up configurations for your Web3 Function run
   w3f: {
     rootDir: "./web3-functions",
     debug: false,
-    networks: ["mumbai", "goerli", "baseGoerli"],
+    networks: ["amoy", "goerli", "baseGoerli"],
```

Comment:
goerli and baseGoerli are both deprecated as well, let's replace them with sepolia instead

Reply:
Fixed